lib/shopify_api/utils/session_utils.rb

```diff
@@ -81,9 +81,10 @@ def current_session_id(auth_header, cookies, online)
           if Context.embedded?
             if auth_header
               matches = auth_header.match(/^Bearer (.+)$/)
-              # add log here
-              raise Errors::MissingJwtTokenError,
-                "Missing Bearer token in authorization header" unless matches
+              unless matches
+                ShopifyAPI::Logger.info("Missing Bearer token in authorization header")
```

Comment:
We wanted to add this because the error gets rescued later on

Reply:
It feels a bit strange to me that a raised error is translated into a log level `INFO`? I'd expect a `warn` at least if not more no?